mqtt_as/mqtt_as.py

```diff
@@ -547,6 +556,8 @@ def __init__(self, config):
         except Exception:
             self.close()
             raise
+        self.dprint("rcv_pids before pruning", self.rcv_pids)
+        self.rcv_pids = set()
```

Comment:
Reinitializing rcv_pids on connect as old pids won't receive any ACK after reconnect. Publish and Subscribe operations will only start adding pids once connecting is established (self._isconnected is True) so won't be adding pids on reconnect before the rcv_pids set has been reinitialized.